Eastern-Research-Group/mywaterway
# Feature/744 remove summarize y

## Related Issues:
* [HMW-744](https://jira.epa.gov/browse/HMW-744)

## Main Changes:
* Removed the `summarize=Y` parameter from calls to the `actions` service. 
  * This was done to improve performance, since Attains is unlikely to resolve this issue any time soon.
  * We also use `summarize=Y` for one of our calls to the `assessments` service and for the `plans` service. In these instances `summarize=Y` actually does improve performance.
  * I made the timeout 2 minutes for `actions` service calls, as some locations (Wisconsin) take about a minute to run (used to take about 5 minutes).
* Fixed an issue where clicking on a waterbody, while on the `Restoration Plans` tab or `Protection Projects` accordion with surrounding waterbodies on, would result in the normal waterbody report. These locations should be showing a popup with a list of plans/actions associated with the waterbody.

## Steps To Test:
1. Open the network tab in chrome dev tools
2. Navigate to http://localhost:3000/waterbody-report/DOEE/DCTFS01R_00/2022
3. Verify the `actions` services return in a timely manner
4. Verify the `Plans to Restore Water Quality` section loads with the `Impairments` column filled out
5. Navigate to http://localhost:3000/community/dc/restore
6. Click on the `Restoration Plans` tab
7. Click a blue waterbody on the map
8. Verify the popup loads with a list of Restoration Plans
9. Click a surrounding waterbody on the map
10. Verify the normal waterbody popup is shown
11. Navigate to http://localhost:3000/community/070700030302/protect
12. Expand the `Protection Projects` accordion
13. Click a blue waterbody on the map
14. Verify the popup loads with a list of Protection Plans
    * This one will take longer as the Wisconsin plans have a over a thousand associated waters
15. Click a surrounding waterbody on the map
16. Verify the normal waterbody popup is shown
